README.md

```diff
@@ -12,75 +12,130 @@ The intent/goal here is that the upstream code in the primary branch for all rep
-- The second job, `display-job-matrix-parameters`, is a simple one that just reports the information that was obtained in `generate-job-matrix-parameters` in case it is needed for debugging.
-- The third job, `crucible-ci-github-runners`, and the forth job, `crucible-ci-self-hosted-runners`, use the information from `generate-job-matrix-parameters` in a "stategy" to generate (potentially large) matrices of test combinations to execute.  Each test that is generated creates a unique job that runs the [integration-tests](README.md#integration-tests) action with the appropriate parameters.  All of the jobs that are created here are queued for execution by the GitHub runners (GitHub hosted or self-hosted depending on the job) and then are executed according to their scheduler and the applicable [usage limits](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits) (currently our GitHub plan is the Free tier).
-- The fifth job, `crucible-ci-build-controller`, is a single targeted job that tests the Crucible controller build logic.
-- The sixth job, `crucible-ci-complete`, executes when all of the jobs created by `crucible-ci-github-runners`, `crucible-ci-self-hosted-runners`, and `crucible-ci-build-controller` complete.  This is the job that is gating the ability to merge the pull request through GitHub's [branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging).
+#### Reusable Workflows
 
-The general workflow looks like this when it runs to completion without any failures:
+These workflows can be called from any Crucible subproject to easily provide a detailed CI test based on provided input.
 
-![GitHub workflow example](docs/crucible-ci.png)
+##### benchmark-crucible-ci
+
+The [benchmark-crucible-ci](.github/workflows/benchmark-crucible-ci.yaml) workflow is intended to be called from Crucible benchmark subprojects.  It limits it's testing focus to benchmark specific jobs.
+
+##### core-crucible-ci
+
+The [core-crucible-ci](.github/workflows/core-crucible-ci.yaml) workflow is intended to be called by any of the core Crucible subprojects.  Some of the core subprojects required more detailed testing (such as controller building under certain conditions) which is implemented in this workflow.
+
+#### Runnable Workflows
+
+These workflows are what run as part of pull requests that are submitted against the Crucible-CI subproject itself.
+
+##### faux-crucible-ci
+
+The [faux-crucible-ci](.github/workflows/faux-crucible-ci.yaml) workflow is a "fake" workflow that is run when the pull request in question does not actually affect runtime behavior (such as documentation only changes).  It provides "null" jobs that satisfy Crucible-CI's GitHub branch protection rules.
+
+##### run-core-crucible-ci
```

Comment:
why do we need core-crucible-ci and run-core ?

Reply:
The reusable workflow (`core-crucible-ci.yaml`) does not execute on it's own, it must be called from something to get it to execute.  The `run-core-crucible-ci.yaml` workflow is that caller and is a very simple example of how other subprojects will be calling the reusable workflows.